### PR TITLE
Feat/create art in gallery v2

### PR DESCRIPTION
**Pull request by yaninavoda**
## Summary of change

1. fixed mistakes pointed at by Olha
2. answered about the StreetcodeId necessity
3. added transaction
4. I made CreateArtResponseDto a mutable record - I need it to be mutable.
ab

### CODE REVIEW

File: Streetcode/Streetcode.BLL/Mapping/Media/Images/ArtProfile.cs [hunk 1]
```diff
@@ -10,5 +9,7 @@ public class ArtProfile : Profile
     public ArtProfile()
     {
         CreateMap<Art, ArtDto>().ReverseMap();
+        CreateMap<Art, CreateArtResponseDto>().ReverseMap();
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.BLL/Mapping/Media/Images/ArtProfile.cs:12]: Not necessary using Reverse
 CreateMap<Art, CreateArtResponseDto>();
 CreateMap<CreateArtRequestDto, Art>();

File: Streetcode/Streetcode.BLL/DTO/Media/Art/CreateArtRequestDto.cs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+﻿namespace Streetcode.BLL.Dto.Media.Art;
+
+public record CreateArtRequestDto(int ImageId, int StreetcodeId, string? Title, string? Description);
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.BLL/DTO/Media/Art/CreateArtRequestDto.cs:3]: is it necessary use int StreetcodeId field?

File: Streetcode/Streetcode.BLL/MediatR/Media/Art/Create/CreateArtRequestDtoValidator.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+namespace Streetcode.BLL.MediatR.Media.Art.Create;
+
+public class CreateArtRequestDtoValidator : AbstractValidator<CreateArtRequestDto>
+{
+    private readonly int _maxTitleLength;
+    private readonly int _maxDescriptionLength;
+
+    public CreateArtRequestDtoValidator()
+    {
+        _maxTitleLength = 150;
```
Comment (OlhaPivniuk) [inline Streetcode/Streetcode.BLL/MediatR/Media/Art/Create/CreateArtRequestDtoValidator.cs:13]: Propose use 
private const int MAXTITLELENGTH = 150;
private const int MAXDESCRPTIONLENGTH = 400;
just like in the tests

File: Streetcode/Streetcode.BLL/DTO/Media/Art/CreateArtRequestDto.cs (hunk 2 above)
Comment (yaninavoda) [reply Streetcode/Streetcode.BLL/DTO/Media/Art/CreateArtRequestDto.cs:3]: as I understood this task we are adding an art to the art GALLERY of some SPECIFIC streetcode. that's why I think StreetcodeId is necessary here

File: Streetcode/Streetcode.BLL/MediatR/Media/Art/Create/CreateArtRequestDtoValidator.cs (hunk 3 above)
Comment (yaninavoda) [reply Streetcode/Streetcode.BLL/MediatR/Media/Art/Create/CreateArtRequestDtoValidator.cs:13]: ok

**Review by HlibBondarev — APPROVED** [review]
Well done!

**Review by OlhaPivniuk — APPROVED** [review]

**Pull request merged**